_settings/settings-local.py

```diff
@@ -22,34 +22,50 @@
     'django_user_agents.middleware.UserAgentMiddleware',
 )
 
-DEBUG = True
-USE_TZ = True
-
 TIME_ZONE = 'America/Vancouver'
 
-################ LOCAL MYSQL ##################
-# DATABASES = {
-#     'default': {
-#         'ENGINE': 'django.db.backends.mysql',
-#         'NAME': 'ubyssey',
-#         'USER': 'root',
-#         'PASSWORD': 'ubyssey',
-#         'HOST': '127.0.0.1',
-#         'PORT': '3306',
-#     },
-# }
+DEBUG = True
+USE_TZ = True
 
+DEV_OPTIONS = ['LOCAL_DOCKER', 'LOCAL', 'PRODUCTION']
+DEV_ENV = DEV_OPTIONS[0]
```

Comment:
Maybe we can provide in a comment available options what the available settings are. `ENVIRONMENT` might be more clear about what this actually means
```python
# ['LOCAL_DOCKER', 'LOCAL', 'PRODUCTION']
ENVIRONMENT = 'LOCAL_DOCKER'
```

Reply:
Yea I like `ENVIRONMENT` better